docs/RFCs/private-network.md

```diff
@@ -0,0 +1,66 @@
+- Feature Name: private-network
+- Start Date: 2022-02-01
+- RFC PR:
+- Functionland Issue:
+
+## Background
+we are using IPFS as our file system. but IPFS is build to use for public data, and it does not support ACL, 
+so we need to find a way to keep user safe until our security layer become mature.
+
+## Problem Statement
+We need to protect user and his/her data from harms and risk of public network.  
+
+## Motivation
+Isolating User from public network can help us relax the problem and build useful product and testing our security layer without putting user in harms way. 
+
+## Proposal
+using IPFS build-in private network. base on [spec](https://github.com/libp2p/specs/blob/master/pnet/Private-Networks-PSK-V1.md)
+Its uses a pre-shared key (PSK) to create a private network with encrypted communication.
+
+For box setup user provide an environment variable SECRET which is password he should remember.
+the secret then convert to a hash of 256 bit by algorithm like sha256 and generate the swarm.key for ipfs and libp2p node's.
+
+For Fula client when user call createClient he should also provide the password he used for setting the box's.
+
+For network discovery its manual process that user should provide all box peer id's in config.
+
+## Scope of work
+- human friendly password to swarm.key 
+- box should look for SECRET env variable and if it exit set the connProtector for libp2p config
+- fula-client constructor should get another parameter call secret and if exist set connProtector for libp2p
+- create an example for the describing functionality
```

Comment:
For me the items listed here sound like "Actionables" more than scope of work. I'd expect a section titled as such to tell the reader to what extent the solutions documented by this RFC solve the bigger problem, along with list of things that are _not_ solved or any simplifying assumptions made.